### DISCUSSION TITLE
Ollama response freeze and get stuck midway in generation on Windows WSL Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by eyeofastarte** [open-webui/open-webui#733] [General]
# Bug Report
Record for an annoying response freezing issue on Window WSL that took me days to solve.

## Description
Responses sometimes work and sometimes get stuck midway.

**Fix**
Simply add '\\wsl.localhost\Ubuntu-22.04' to your antivirus exclusion
https://github.com/microsoft/WSL/issues/8824#issuecomment-1253204941

## Environment
OS: Windows 11, WSL 2, Ubuntu 22.04

## Reproduction Details
Standard installation of everything + run:
sudo apt update
sudo apt upgrade

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I have reviewed the troubleshooting.md document.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Additional Information
Not an actual fix and does not get to the root of the issue, but it works, so maybe a fix can be made in the future.

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Thanks for reporting this issue, I'll move this post to discussion for better visibility!

**Comment by narmaku** [comment]
Hello, this also happens on Linux, Fedora 39. I have already experienced this issue several times on my local Open-webUI setup. It does not always happen, but it does happen quite often.

Setup details:
- Installation type: local, docker container
- Model: llama3:7b (but it does happen with other models too, always using ollama)
- Browser: Google Chrome 125.0.6422.141 (Official Build) (64-bit)
- Installed browser add-on: uBlock Origin
- Resources: CPU-only, 8 threads, 32 GB of RAM.

The "stop" button is not working either. Ollama processes are done, which indicates that the processing of the reply finished in the ollama backend, but not rendered correctly or not received by open-webui.

When I reload the page, the "incomplete" response disappears, along with my previous reply. 

![image](https://github.com/open-webui/open-webui/assets/100693208/47a74db5-485a-458b-93e5-58090ae03ec2)

I can reproduce this multiple times, so if you want me to gather some more logs let me know and, if possible, please provide me with further instructions.

Thanks!

**Comment by vishnunuk** [comment]
Same issue found i used 
deepseek-coder-v2:latest

**Comment by mattigins** [comment]
I am also experiencing this issue. Also using deepseek-coder-v2:latest

**Comment by Captain-Pumpkinhead** [comment]
I have this issue with any model Qwen or DeepSeek based.

Installation: Docker Compose
OS: Windows 11, WSL (Ubuntu)
GPU: AMD RX 7900XTX
